src/c_api.rs

```diff
@@ -407,26 +338,29 @@ pub fn json_api_get<M: Manager>(
         return null();
```

Comment:
Do you think it would matter (slightly improve performance) if we take the optimistic approach, and assume less branch-none-taken will occur when we start by testing for the successful option that the path exists,
And only otherwise handle the failed option that the path does not exist and return null?

Reply:
This does not verify that the path exists, it verifies that the given path is a valid JSON path...